andela/ah-sealteam
# [feature #161278908] Email verification link on user sign up

### What does this PR do
This test if a user can receive a verification link on sign up and 
User clicks on that link to verify their account 

### Description of Task to be completed?

1. When a user signs up on our platform, he or she should receive an email verification link.
2. When the user clicks on the link provided, he or she should be redirected to verify account page
3. When the user doesn't click the link within the time set, the link expires in which they will need to request for a new link.

### How should this be manually tested?*
On your git repo run
$ `git fetch origin pull/22/head:ft-user-email-verification-161278908`
checkout to the feature branch
$ `git checkout  ft-user-email-verification-161278908`
> register a new user on the postman
> Go to your email and verify your account by clicking the link
> You should see a successfully verified message
> After 30 seconds the ur
### Screenshot(s) available
Click link to verify account
<img width="755" alt="screen shot 2018-11-08 at 10 18 49" src="https://user-images.githubusercontent.com/8876906/48183469-d02f3680-e33f-11e8-95c1-1716bc76fdf0.png">
Verify account page
<img width="1152" alt="screen shot 2018-11-08 at 10 17 21" src="https://user-images.githubusercontent.com/8876906/48183524-053b8900-e340-11e8-8fd5-dedb283fc9b4.png">



### What are the relevant pivotal tracker stories?
[161278908](https://www.pivotaltracker.com/story/show/161278908)
